src/org/usfirst/frc/team2706/robot/commands/InitLift.java

```diff
@@ -0,0 +1,11 @@
+package org.usfirst.frc.team2706.robot.commands;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class InitLift extends CommandGroup {
+
+    public InitLift() {
+        this.addSequential(new SetLiftHeightBlocking(0.25,2,0.1),0.25);
```

Comment:
These double values should probably be constants in RobotMap.    That would better explain what the code is doing.